example/dcgan/README.md

```diff
@@ -27,22 +31,34 @@ The `dcgan` binary will be installed at `$GOPATH/bin` directory.
 Then, we execute the compiled binary in current directory to train the model:
 
 ```bash
-$GOPATH/bin/dcgan -dataroot=$DATAROOT
-[0/100][0] D_Loss: 1.501880 G_Loss: 3.284637
-[0/100][1] D_Loss: 1.180851 G_Loss: 3.828784
-[0/100][2] D_Loss: 0.901259 G_Loss: 4.031792
-[0/100][3] D_Loss: 0.868884 G_Loss: 4.756028
-[0/100][4] D_Loss: 0.832174 G_Loss: 5.523646
-[0/100][5] D_Loss: 0.762144 G_Loss: 5.824744
-[0/100][6] D_Loss: 0.488496 G_Loss: 6.646035
-[0/100][7] D_Loss: 0.381518 G_Loss: 6.663010
-[0/100][8] D_Loss: 0.626897 G_Loss: 9.005035
-[0/100][9] D_Loss: 0.393866 G_Loss: 6.695397
+$GOPATH/bin/dcgan -dataroot=$DATAROOT/train.tgz
+[0/10][0] D_Loss: 1.384754 G_Loss: 3.605948
+[0/10][1] D_Loss: 1.666008 G_Loss: 5.663126
+[0/10][2] D_Loss: 0.599109 G_Loss: 6.945201
+[0/10][3] D_Loss: 0.215684 G_Loss: 6.853242
+[0/10][4] D_Loss: 0.599603 G_Loss: 5.772169
+[0/10][5] D_Loss: 0.427933 G_Loss: 7.049569
+[0/10][6] D_Loss: 0.703852 G_Loss: 7.072733
+[0/10][7] D_Loss: 0.365140 G_Loss: 7.365425
+[0/10][8] D_Loss: 0.193228 G_Loss: 7.702262
+[0/10][9] D_Loss: 0.344985 G_Loss: 7.280960
 ```
 
 The training program periodically generates image samples and saves to pickle files.
-We use the `pickle_to_png.py` to transform the saved pickle files into png format.
+We provide a script to transform the saved pickle files into png format.
+
+```bash
+python visualize_pickle.py --save_image=1
+```
+
+Here are some generated images after 10 epoches training:
+
+![example1](1.png) ![example2](2.png) ![example3](3.png)
+
+And the script could also generate an animation to visualize
+the training progress of generated fake images.
+`ffmpeg` is needed to save the animation to mp4 format.
 
 ```bash
-python pickle_to_png.py
+python visualize_pickle.py --save_video=1
```

Comment:
Should we have a Go utility function to visualize a gotorch.Tensor into an image?

Reply:
I believe we should have.  And I think that it's not so urgent to add it right now. We could do this later.